Domain/Models/City.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Models
+{
+    public class City : BaseEntity
+    {
+        public string Name { get; set; }
+    }
```

Comment:
It should contain the Icollection of related models

Reply:
we have separate task for adding relations between  tables after all models are done, so can you approve this PR?